docs/preview/01-index.md

```diff
@@ -17,13 +17,13 @@ PM > Install-Package Arcus.Messaging.Abstractions
 
 # Features
 
-- Support for using message pumps for the following brokers:
-    - Azure Service Bus ([docs](./02-Features/message-pumps/service-bus.md) | [extensions](./02-Features/service-bus.md))
-- Support for exposing TCP health probes to periodically check liveness/readiness of the host ([docs](./02-Features/tcp-health-probe.md))
-- Customize message pumps ([docs](./02-Features/message-pumps/customization.md))
-    - Fallback message handlers
-- Using message pumps with Azure Functions
-    - Azure Service Bus ([docs](./02-Features/message-pumps/service-bus-azure-functions.md))
+- Support for using message handling for the following brokers:
+  - Azure Service Bus
+    - Worker ([docs](./02-Features/02-message-handling/01-service-bus.md))
```

Comment:
What exactly is meant by 'worker'.  This is actually a background worker that processes messages from the bus.
I wonder if we can come up with a better name.

Reply:
Yeah, like we did in templates, we make a distinction between .NET SDK Workers and Azure Functions. So, seemed that we should keep this distinction here, too.

Reply:
I agree that there must be a distinction, but I was just thinking if we could come up with a better name then just 'worker'.  (Background Worker, i dunno)

Reply:
Maybe 'Worker Service`, then? Like Microsoft describes them: https://docs.microsoft.com/en-us/dotnet/core/extensions/workers

Reply:
Indeed, that would be great.

Reply:
Done in https://github.com/arcus-azure/arcus.messaging/pull/268/commits/6d546c732af7afb94c39e8d3e190773988961da5